PlaidLink/Classes/PLDLinkBankMFALoginViewController.m

```diff
@@ -26,6 +27,18 @@ - (instancetype)initWithInstitution:(PLDInstitution *)institution
   if (self = [super init]) {
     _institution = institution;
     _product = product;
+    _options = [NSMutableDictionary new];
```

Comment:
Remove, also party our fault for not documenting our style explicitly but we use alloc/init vs. new in this code base